Description: E2fsprogs 1.43.4 release fixup
 The orig tar.gz that was uploaded to e2fsprogs 1.43.4-1 was not the
 final release tarball; it had an outdated version of e2fsprogs.lsm
 and e2fsprogs.spec.  This doesn't affect generated binary packages of
 e2fsprogs at all, but let's correct it to make it easier generate
 updated packages.
From: Theodore Y. Ts'o <tytso@mit.edu>

diff -ru e2fsprogs-1.43.4/e2fsprogs.lsm /tmp/e2fsprogs-1.43.4/e2fsprogs.lsm
--- e2fsprogs-1.43.4/e2fsprogs.lsm	2017-01-31 17:58:16.000000000 -0500
+++ /tmp/e2fsprogs-1.43.4/e2fsprogs.lsm	2017-01-31 19:58:05.000000000 -0500
@@ -8,8 +8,8 @@
 Author:         tytso@mit.edu (Theodore Tso)
 Maintained-by:  tytso@mit.edu (Theodore Tso)
 Primary-site:   ftp.kernel.org /pub/linux/kernel/people/tytso/e2fsprogs
-                7224kB e2fsprogs-1.43.4.tar.gz
-                644kB  e2fsprogs-libs-1.43.4.tar.gz
+                7376kB e2fsprogs-1.43.4.tar.gz
+                588kB  e2fsprogs-libs-1.43.4.tar.gz
                 1kB    e2fsprogs-1.43.4.lsm
 Alternate-site: download.sourceforge.net /pub/sourceforge/e2fsprogs
 Platforms:	linux 1.2.x/1.3.x/2.0.x/2.1.x/2.2.x/2.3.x/2.4.x/2.5.x/2.6.x/3.x/4.x
diff -ru e2fsprogs-1.43.4/e2fsprogs.spec /tmp/e2fsprogs-1.43.4/e2fsprogs.spec
--- e2fsprogs-1.43.4/e2fsprogs.spec	2017-01-31 17:46:32.000000000 -0500
+++ /tmp/e2fsprogs-1.43.4/e2fsprogs.spec	2017-01-31 20:03:06.000000000 -0500
@@ -5,7 +5,7 @@
 
 Summary: Utilities for managing ext2/ext3/ext4 filesystems
 Name: e2fsprogs
-Version: 1.43.4~WIP.2017.01.30
+Version: 1.43.4
 Release: 0
 License: GPLv2
 Group: System Environment/Base
